ner_v1/detectors/textual/name/name_detection.py

```diff
@@ -319,7 +320,7 @@ def replace_detected_text(self, text_detection_result, text):
             replaced_text_tokens = nltk_tokenizer.tokenize(text.lower())
         elif self.language in EUROPEAN_LANGUAGES_SET:
             replaced_text_tokens = spacy_utils.tokenize(text.lower())
-        elif self.language in INDIC_LANGUAGES_SET:
+        elif self.language in INDIC_LANGUAGES_SET.union(OTHER_AVAILABLE_LANGUAGES):
```

Comment:
Use simple if condition
`elif self.language in INDIC_LANGUAGES_SET or self.language in OTHER_AVAILABLE_LANGUAGES:`